compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -639,7 +640,8 @@ static LogicalResult setRootConfigImpl(mlir::FunctionOpInterface entryPointFn,
         // let it first crash for all the other ops and then consiously
         // add support for them, this way we can verify our work.
         // TODO (vivian): add support for other conv interface ops
-        .Case<linalg::Conv2DNhwcHwcfOp, linalg::Conv2DNchwFchwOp>([&](auto op) {
+        .Case<linalg::Conv2DNhwcHwcfOp, linalg::Conv2DNchwFchwOp,
+              linalg::Conv2DNhwcHwcfQOp>([&](auto op) {
```

Comment:
I don't think we have lit test for Convs in lowering_strategy.mlir - it'd be nice to add at least one ConvOp there (if not present).

Reply:
Actually I added a test for conv2d lowering strategy in my previous PR. https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lowering_strategy_conv.mlir